scanpy/testing/_pytest/__init__.py

```diff
@@ -55,9 +55,26 @@ def limit_multithreading():
         max_threads = max(n_cpus // n_workers, 1)
 
         with threadpoolctl.threadpool_limits(limits=max_threads):
-            yield
+            yield max_threads
```

Comment:
What's this about?

Reply:
Fixtures that don’t start with an underscore return things. I just made this one return a thing.

I can alternatively add an underscore to its name